src/main/scala/org/broadinstitute/dsde/firecloud/Boot.scala

```diff
@@ -98,7 +98,7 @@ object Boot extends App with LazyLogging {
     // can be disabled
     val agoraDAO: AgoraDAO = whenEnabled[AgoraDAO](FireCloudConfig.Agora.enabled, new HttpAgoraDAO(FireCloudConfig.Agora))
     val googleServicesDAO: GoogleServicesDAO = whenEnabled[GoogleServicesDAO](FireCloudConfig.GoogleCloud.enabled, new HttpGoogleServicesDAO(FireCloudConfig.GoogleCloud.priceListUrl, GooglePriceList(GooglePrices(FireCloudConfig.GoogleCloud.defaultStoragePriceList, UsTieredPriceItem(FireCloudConfig.GoogleCloud.defaultEgressPriceList)), "v1", "1")))
-    val importServiceDAO: ImportServiceDAO = whenEnabled[ImportServiceDAO](FireCloudConfig.ImportService.enabled, new HttpImportServiceDAO)
+    val importServiceDAO: ImportServiceDAO = whenEnabled[ImportServiceDAO](FireCloudConfig.ImportService.enabled, new HttpImportServiceDAO(FireCloudConfig.ImportService.enabled))
```

Comment:
I don't think we can use `FireCloudConfig.ImportService.enabled` as an argument to `HttpImportServiceDAO`, because if `FireCloudConfig.ImportService.enabled` is false, then `HttpImportServiceDAO` won't be used. It will be replaced with the result of `DisabledServiceFactory.newDisabledService`.

Reply:
This is true, and it makes some of the code in this PR pointless. BUT, the end result is fine. `DisabledServiceFactory.newDisabledService` returns a proxy whose `isEnabled` method returns false and throws an error for other methods. All we care about is that `isEnabled` returns false, because that short-circuits everything else.

Unit tests pass, swat tests pass, my manual testing in a bee looks good

Reply:
Ok. I think I was getting this mixed up with Rawls, where ImportService is higher level and talks to both Import Service and CWDS.

Reply:
☝️